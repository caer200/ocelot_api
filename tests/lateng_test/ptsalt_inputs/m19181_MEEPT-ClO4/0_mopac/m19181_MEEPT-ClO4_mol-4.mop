CHARGES

 
Cl	1.503419 +1 12.617463 +1 5.618322 +1
O	1.007620 +1 13.950481 +1 5.440607 +1
O	0.420016 +1 11.676047 +1 5.460402 +1
O	2.065739 +1 12.480548 +1 6.933638 +1
O	2.517029 +1 12.358466 +1 4.634987 +1
