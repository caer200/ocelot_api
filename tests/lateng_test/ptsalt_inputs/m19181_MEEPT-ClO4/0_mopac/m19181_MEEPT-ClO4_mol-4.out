 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:09:21 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Cl          1.50341900  *  12.61746300  *   5.61832200  *
     2       O          1.00762000  *  13.95048100  *   5.44060700  *
     3       O          0.42001600  *  11.67604700  *   5.46040200  *
     4       O          2.06573900  *  12.48054800  *   6.93363800  *
     5       O          2.51702900  *  12.35846600  *   4.63498700  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Cl          1.5034     12.6175      5.6183
     2         O          1.0076     13.9505      5.4406
     3         O          0.4200     11.6760      5.4604
     4         O          2.0657     12.4805      6.9336
     5         O          2.5170     12.3585      4.6350

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: O4 Cl  =     5 atoms



      MOLECULAR POINT GROUP   :   C1  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1         Cl      2      3      4      5
      2          O      1
      3          O      1
      4          O      1
      5          O      1

     Ion Atom No.  Type    Charge
 
    1       1     Cl       +1
 
    1       2      O       -1
    2       3      O       -1
    3       4      O       -1
    4       5      O       -1

          COMPUTED CHARGE ON SYSTEM: -3

          COMPUTED CHARGE ON SYSTEM:  -3

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
